lib/game_end_evaluator.rb

```diff
@@ -1,11 +1,11 @@
 class GameEndEvaluator
-  def find_winner(board)
+  def player_won?(board)
     [board.rows, board.columns, board.diagonals].each do |board_state|
       winner = find_winner_in_matrix board_state
```

Comment:
Could this method have an interface that requires less conditional checking?

Reply:
I wasn't 100% sure what you meant by interface, but I did try to reduce conditional checking.

Reply:
This was just a small comment on creating method interfaces that accomplish what their results are being used for. Finding the winner does not seem to be what this method's result is being used to accomplish. It seems that the result of the method is just to find out if a game is won. Instead of returning a winner or nil that then needs to be checked in order to return whether or not the game has been won, maybe the method could just return that boolean state?

Reply:
I think this would be something that we could pair on. This is not a blocker.